openssl/README.md

```diff
@@ -13,3 +13,82 @@ Binary package
 ## Usage
 
 *TODO: Add instructions for usage*
+
+## Stage 1 Plan
+
+This section outlines a general playbook for maintainers who are building this plan.
+
+As a stage 1 plan, this plan has extra considerations when merging PRs and building the plan. This stage 1 plan can safely be built without a core plans base refresh.
+
+1. The following plans must be verified as successfully building, along with success of any automated tests, before merging any PRs:
+
+```
+core-plans/openssl
+core-plans/openssl-musl/
+habitat/components/hab/
```

Comment:
`habitat` origin plans are out of scope for core-plans.

Reply:
Sorry this is confusing. hab-sup and some of the other habitat components are built in the core origin here, these are repo references. I'll fix the confusion.